src/components/sourceCard/SourceCard.tsx

```diff
@@ -77,25 +87,58 @@ export default function SourceCard({
           onBlur={saveText}
         />
       </div>
-      <SourceThumbnail source={source} src={src} />
-      <h2 className="absolute bottom-0 text-p text-xs bg-zinc-900 w-full bg-opacity-90">
-        {t('source.ingest', {
-          ingest: source.ingest_name
-        })}
-      </h2>
-      <button
-        className="absolute bottom-0 right-0 text-p hover:border-l hover:border-t bg-red-700 hover:bg-red-600 min-w-fit p-1 rounded-tl-lg"
-        onClick={() => {
-          onSourceRemoval({
-            _id: source._id.toString(),
-            label: source.label,
-            input_slot: source.input_slot,
-            stream_uuids: source.stream_uuids
-          });
-        }}
-      >
-        <IconTrash className="text-p w-4 h-4" />
-      </button>
+      {source && source.src && (
+        <SourceThumbnail source={source} src={src} type={type} />
```

Comment:
This has been changed to use the ImageComponent. Have you merged main into this branch?